assets/js/components/KeyMetrics/MetricsSelectionPanel/index.test.js

```diff
@@ -134,6 +134,50 @@ describe( 'MetricsSelectionPanel', () => {
 			).toHaveTextContent( 'Top performing keywords' );
 		} );
 
+		it( 'should list module disconnect notice required modules are disconnected', () => {
+			provideKeyMetrics( registry );
+
+			provideModules( registry, [
+				{
+					slug: 'analytics-4',
+					active: false,
+					connected: false,
+				},
+				{
+					slug: 'search-console',
+					active: false,
+					connected: false,
+				},
+			] );
+
+			provideKeyMetricsWidgetRegistrations( registry, {
+				[ KM_SEARCH_CONSOLE_POPULAR_KEYWORDS ]: {
+					modules: [ 'search-console' ],
+				},
+				[ KM_ANALYTICS_LOYAL_VISITORS ]: {
+					modules: [ 'analytics-4' ],
+				},
+			} );
+
+			render( <MetricsSelectionPanel />, { registry } );
+
+			expect(
+				document.querySelector(
+					'.googlesitekit-km-selection-panel-metrics'
+				)
+			).toHaveTextContent(
+				'Analytics 4 is disconnected, no data to show'
+			);
+
+			expect(
+				document.querySelector(
+					'.googlesitekit-km-selection-panel-metrics'
+				)
+			).toHaveTextContent(
+				'Search Console is disconnected, no data to show'
+			);
+		} );
+
```

Comment:
Let's add another test for the text content when multiple modules are disconnected.

Reply:
@tofumatt this has been addressed in https://github.com/google/site-kit-wp/pull/7604/commits/d15ea0261cb95e0de1abd078b2f62658a0cae11c